### DISCUSSION TITLE
What is the name of the dog in the Garfield comic strip?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#581] [Galaxy Trivia] [answered]
**Bonus:** And what is its owner's name?

**Comment by peterbaccaro** [comment, marked as answer]
**Odie** is the name of the dog who appears in the comic strip Garfield by Jim Davis.
The name of the current owner of Odie is **Jon Arbuckle**.
The former owner of Odie was Lyman who was a friend and roommate of Jon Arbuckle.  Lyman is said to have left Odie with Jon, before moving away to work as a wildlife photographer in a distant jungle.

![odie](https://user-images.githubusercontent.com/74666079/232544686-56b38621-fc55-468f-8b85-c4de786cf238.png)

**Comment by thuongtruong1009** [comment]
The name of the dog in the Garfield comic strip is Odie, and its owner's name is Jon Arbuckle.
